crates/rattler_solve/src/libsolv/wrapper/repo.rs

```diff
@@ -306,12 +307,15 @@ impl RepoRef {
             } else {
                 tracing::warn!("unknown package extension: {}", &repo_data.file_name);
             }
+
+            // Store the solvable id
+            solvable_ids.push(solvable_id);
         }
 
         // TODO: What does this do?
         unsafe { ffi::repo_internalize(self.as_ptr().as_ptr()) };
```

Comment:
Do we know what this does already?

Reply:
I think that will internalize all the strings into the big libsolv Pool or something like that